src/app/chat/views/messages.nim

```diff
@@ -54,6 +54,7 @@ QtObject:
     result.communities = communitiesView
     result.messageList = initOrderedTable[string, ChatMessageList]()
     result.pinnedMessagesList = initOrderedTable[string, ChatMessageList]()
+    //TODO should this be removed?
     result.messageList[status_utils.getTimelineChatId()] = newChatMessageList(status_utils.getTimelineChatId(), result.status, false)
```

Comment:
Yep, I think that should be removed too!

Reply:
done

> I think more QML can be removed I think the way we display message is different for timeline and as such can be removed too. I believed this is this:
> https://github.com/status-im/status-desktop/blob/master/ui/imports/shared/views/chat/MessageView.qml#L330-L356
> (but please double check ;))

Checking that I see that `chatType: 4` is  `Profile` :
```
type ChatType* {.pure.}= enum
  Unknown = 0,
  OneToOne = 1, 
  Public = 2,
  PrivateGroupChat = 3,
  Profile = 4,
  
```
could somebody confirm that `Profile` corresponds to readonly `property int chatTypeStatusUpdate: 4` from Contants.qml? @PascalPrecht git blames you for the implementation, commit `f512d5063c5853dcdee9670e17defbcde4275fbe` "feat: introduce StatusUpdateChatInput variation"